Code/Engine/Foundation/Math/Implementation/Easing.cpp

```diff
@@ -0,0 +1,53 @@
+
+#include <Foundation/Math/Easing.h>
+
+#include <unordered_map>
+
+namespace ezMath
+{
+  ezEasingFunction GetEasingFunction(ezEasingFunctions easingFunction)
+  {
+    static std::unordered_map<ezEasingFunctions, ezEasingFunction> easingFunctions;
```

Comment:
No need for the lookup table. A switch statement would be even more efficient.

Reply:
Resolved.